alicloud/resource_alicloud_alikafka_consumer_group_test.go

```diff
@@ -0,0 +1,176 @@
+			return alikafkaClient.GetConsumerList(request)
+		})
+		if err != nil {
+			log.Printf("[ERROR] Failed to retrieve alikafka consumer groups on instance (%s): %s", instanceId, err)
+			continue
+		}
+
+		consumerListResp, _ := raw.(*alikafka.GetConsumerListResponse)
+		consumers := consumerListResp.ConsumerList.ConsumerVO
+		for _, v := range consumers {
+			name := v.ConsumerId
+			skip := true
+			for _, prefix := range prefixes {
+				if strings.HasPrefix(strings.ToLower(name), strings.ToLower(prefix)) {
+					skip = false
+					break
+				}
+			}
+			if skip {
+				log.Printf("[INFO] Skipping alikafka consumer id: %s ", name)
+				continue
+			}
+			log.Printf("[INFO] delete alikafka consumer group: %s ", name)
+
+			// Control the consumer group delete rate
+			time.Sleep(time.Duration(400) * time.Millisecond)
+
+			request := alikafka.CreateDeleteConsumerGroupRequest()
+			request.InstanceId = instanceId
+			request.ConsumerId = v.ConsumerId
+			request.RegionId = defaultRegionToTest
+
+			_, err := alikafkaService.client.WithAlikafkaClient(func(alikafkaClient *alikafka.Client) (interface{}, error) {
+				return alikafkaClient.DeleteConsumerGroup(request)
+			})
+			if err != nil {
+				log.Printf("[ERROR] Failed to delete alikafka consumer group (%s): %s", name, err)
+			}
+		}
+	}
+
+	return nil
+}
+
+func TestAccAlicloudAlikafkaConsumerGroup_basic(t *testing.T) {
+
+	var v *alikafka.ConsumerVO
+	resourceId := "alicloud_alikafka_consumer_group.default"
+	ra := resourceAttrInit(resourceId, alikafkaConsumerGroupBasicMap)
+	serviceFunc := func() interface{} {
+		return &AlikafkaService{testAccProvider.Meta().(*connectivity.AliyunClient)}
+	}
+	rc := resourceCheckInit(resourceId, &v, serviceFunc)
+	rac := resourceAttrCheckInit(rc, ra)
+
+	rand := acctest.RandInt()
+	testAccCheck := rac.resourceAttrMapUpdateSet()
+	name := fmt.Sprintf("tf-testacc-alikafkaconsumerbasic%v", rand)
+	testAccConfig := resourceTestAccConfigFunc(resourceId, name, resourceAlikafkaConsumerGroupConfigDependence)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccPreCheckWithAlikafkaInstanceSetting(t)
+		},
+		// module name
+		IDRefreshName: resourceId,
+		Providers:     testAccProviders,
+		CheckDestroy:  rac.checkResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"instance_id": os.Getenv("ALICLOUD_INSTANCE_ID"),
+					"consumer_id": "${var.consumer_id}",
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(map[string]string{
+						"consumer_id": fmt.Sprintf("tf-testacc-alikafkaconsumerbasic%v", rand),
+					}),
+				),
+			},
+
+			{
+				ResourceName:      resourceId,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"consumer_id": "modified_consumer_id",
```

Comment:
I think there is no need to test `consumer_id` changing because of it is forcenew.